crowdin/model/projects.go

```diff
@@ -243,6 +243,13 @@ type ProjectsAddRequest struct {
 	TMContextType  string                 `json:"tmContextType,omitempty"`
 	TMPreTranslate *ProjectTMPreTranslate `json:"tmPreTranslate,omitempty"`
 	MTPreTranslate *ProjectMTPreTranslate `json:"mtPreTranslate,omitempty"`
+	// FIXME: missing fields for aiPreTranslate and assistActionAiPromptId ( TODO: create new issue )
+	// Translation Memory ID.
+	// Default: null
```

Comment:
I would suggest adding these properties to the current PR and renaming it to a more generic name

Reply:
Added those properties and also added in the test `TestProjectsService_Add`